packages/react-mui-branding/src/components/BrandingThemeProvider/index.tsx

```diff
@@ -34,8 +36,9 @@ const BrandingThemeProvider: FC<BrandingThemeProviderProps> = ({
       const themeOptions_ = getBrandingThemeOptions(
         configProp ?? (config || {})
       );
-      return createTheme({ ...themeOptions_, ...themeOptions });
-    } catch {
+      return createTheme(merge({ ...themeOptions_ }, { ...themeOptions }));
+    } catch (e) {
+      console.error(e);
```

Comment:
Stray `console.log()`?

Reply:
No this was an intentional error, we don't want this throwing an uncaught exception, but we at least want to console error log so it is somewhat visible in development